src/SOS/Strike/util.cpp

```diff
@@ -3345,6 +3345,43 @@ bool IsRuntimeVersion(VS_FIXEDFILEINFO& fileInfo, DWORD major)
     return false;
 }
 
+bool IsRuntimeVersionAtLeast(DWORD major)
+{
+    VS_FIXEDFILEINFO fileInfo;
+    if (GetEEVersion(&fileInfo, nullptr, 0))
+    {
+        return IsRuntimeVersionAtLeast(fileInfo, major);
+    }
+    return false;
+}
+
+bool IsRuntimeVersionAtLeast(VS_FIXEDFILEINFO& fileInfo, DWORD major)
+{
+    switch (major)
+    {
+        case 3:
+            if (HIWORD(fileInfo.dwFileVersionMS) == 4 && LOWORD(fileInfo.dwFileVersionMS) == 700)
+            {
+                return true;
+            }
+            // fall through
+
+        case 5:
+            if (HIWORD(fileInfo.dwFileVersionMS) == 5)
```

Comment:
```suggestion
            if (HIWORD(fileInfo.dwFileVersionMS) >= 5)
```

Reply:
If this were to be acceptable, I would prefer using the alternate solution of replacing with `GetRuntimeVersion`, which is more flexible in usage. I opted against that only because the existing assertions seemed to try to indicate a failure upon an undetected runtime version that is queried, such that if a feature were to be added to a new runtime version and versioned for it, the assertion would fail (with the proposed change, that would no longer be the case). If that assertion is not too important (I don't know but I would guess that any new version of the runtime would naturally entail adding detection of it), then the `GetRuntimeVersion` solution looks far better to me (without the assertion that I mentioned above).

Reply:
The assert would still fail when checking for a newer runtime version until a new case label is added, this change should be fine